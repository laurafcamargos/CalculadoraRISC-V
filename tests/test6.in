2147483647
+
1
-
2147483648
*
2
f
