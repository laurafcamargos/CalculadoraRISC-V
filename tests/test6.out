Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: -2147483648
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 0
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 0
Digite a operação (+, -, *, /, u, f): 
